docs/spec/spec.md

```diff
@@ -834,7 +842,21 @@ resource function get fruit(string? colour, http:Caller caller) {
     return; // 500 internal Server Error
 }
 ```
-#### 2.3.5. Introspection resource
+
+##### 2.3.5.3. Default response status codes
+
+To improve the developer experience for RestFul API development, following default status codes will be used in outbound 
+response when returning `anydata` directly from a resource function.
+
+| Resource Accessor | Semantics                                                     | Status Code             |
+|-------------------|---------------------------------------------------------------|-------------------------|
+| GET               | Let's retrieve the resource                                   | 200 OK                  |
```

Comment:
Shall we remove the `Let's` part?

Reply:
Fixed with [729dac54fbaadc793253fc31f5d5519b34d2b9e7](https://github.com/ballerina-platform/module-ballerina-http/pull/1101/commits/729dac54fbaadc793253fc31f5d5519b34d2b9e7)